action/create_disk.go

```diff
@@ -14,8 +14,8 @@ func NewCreateDisk(diskCreator bslcdisk.Creator) CreateDisk {
 	return CreateDisk{diskCreator: diskCreator}
 }
 
-func (a CreateDisk) Run(size int, _ VMCID) (DiskCID, error) {
-	disk, err := a.diskCreator.Create(size)
+func (a CreateDisk) Run(size int, instanceId VMCID) (DiskCID, error) {
+	disk, err := a.diskCreator.Create(size, int(instanceId))
```

Comment:
Look at what I did for `VMID`, I think there is a way to add the `boxing` and `unboxing` methods so you don't have to do the explicit `int(...)` cast. See the [cids.go](https://github.com/maximilien/bosh-softlayer-cpi/blob/master/action/cids.go). That is likely where you need to add methods for `DiskCID` to convert back and forth to `string` and `int`.